src/Cake.Common/Tools/DotNet/Package/Add/DotNetPackageAdder.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Cake.Core;
+using Cake.Core.IO;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.DotNet.Package.Add
```

Comment:
What do you think, that path is correct? Then we'll able to add [Package.List](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-list-package) and [Package.Remove](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-remove-package) commands

Reply:
@Marusyk Yes, that works.